parser_library/test/processing/db2_preprocessor_test.cpp

```diff
@@ -404,7 +481,21 @@ TEST(db2_preprocessor, include_nonexistent)
     a.analyze();
     a.collect_diags();
 
-    EXPECT_EQ(a.diags().size(), (size_t)1);
+    EXPECT_TRUE(matches_message_codes(a.diags(), { "DB002" }));
+}
+
+TEST(db2_preprocessor, include_invalid)
+{
+    mock_parse_lib_provider libs({
+        { "MEMBER", "" },
+    });
+    std::string input = " EXEC SQL INCLUDEMEMBER ";
+
+    analyzer a(input, analyzer_options { &libs, db2_preprocessor_options {} });
+    a.analyze();
+    a.collect_diags();
+
+    EXPECT_EQ(a.hlasm_ctx().get_visited_files().count(member_loc), 0);
```

Comment:
do we produce a diagnostic, or do we just let it fail in the opencode?

Reply:
- It behaves like if it was any other ` EXEC SQL ` statement with non include operands like ` EXEC SQL op1 op2`

- It produces a bunch of errors coming from opencode